tagcompare/placelocal.py

```diff
@@ -128,7 +130,13 @@ def _get_all_pids(pids):
     return result
 
 
-def get_cids(cids=None, pids=None):
+def get_cids_from_settings():
+    cids = settings.DEFAULT.campaigns
+    pids = settings.DEFAULT.publishers
+    return _get_cids(cids, pids)
```

Comment:
This is a bit magical, I'd expect (without seeing comments clarifying) this to simply return a list of cids (ie return line 134, cid)